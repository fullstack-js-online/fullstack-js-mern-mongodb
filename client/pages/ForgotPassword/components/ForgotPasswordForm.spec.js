import React from 'react'
import { shallow } from 'enzyme'

import ForgotPasswordForm from './ForgotPasswordForm'

describe('ForgotPasswordForm form', () => {
  const props = {
    initialValues: {
      email: '',
      password: ''
    },
    validationSchema: {},
    onSubmit: jest.fn()
  }

  it('should match snapshot', () => {
    expect(shallow(<ForgotPasswordForm {...props} />)).toMatchSnapshot()
  })
})
