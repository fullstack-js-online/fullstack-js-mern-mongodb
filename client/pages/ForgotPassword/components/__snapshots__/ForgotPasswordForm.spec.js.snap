// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForgotPasswordForm form should match snapshot 1`] = `
<Fragment>
  <div
    className="container mx-auto flex justify-center"
  >
    <div
      className="max-w-md-1/2 mx-3 my-12 w-full"
    >
      <h1
        className="text-center font-primary font-semibold text-gold"
      >
        Forgot Password
      </h1>
      <div
        className="bg-white shadow-md my-12 p-16 rounded w-full"
      >
        <Formik
          enableReinitialize={false}
          initialValues={
            Object {
              "email": "",
              "password": "",
            }
          }
          isInitialValid={false}
          onSubmit={[MockFunction]}
          validateOnBlur={true}
          validateOnChange={true}
          validationSchema={Object {}}
        >
          <Component />
        </Formik>
      </div>
    </div>
  </div>
</Fragment>
`;
